src/Computer.js

```diff
@@ -0,0 +1,24 @@
+const MissionUtils = require('@woowacourse/mission-utils');
+const { Random } = MissionUtils;
```

Comment:
개인적인 의견이지만, 따로 변수를 선언해서 구조 분해 해주지 않아도 생각해요!
```suggestion
const { Random } = require('@woowacourse/mission-utils');
```